### PR TITLE
feat: new feature give feedback for events has completed

### PR DESCRIPTION
**Pull request by turrs**
please give review for feature give feedback

### CODE REVIEW

File: src/Parts/MyTrainingCard/SingleTrainingCard.jsx [hunk 1]
```diff
@@ -1,42 +1,57 @@
...
+import { Rate } from "antd";
 
-export const SingleTrainingCard = (props) => {
+export const SingleTrainingCard = props => {
   const { item, id, location } = props;
   const navigate = useNavigate();
 
   const openLocation = () => {
-    console.log('test hit');
+    console.log("test hit");
```
Comment (argorfc) [inline src/Parts/MyTrainingCard/SingleTrainingCard.jsx]: Please remove log!

**Review by argorfc — CHANGES_REQUESTED** [review]
Read the comments!

File: src/Parts/MyTrainingCard/SingleTrainingCard.jsx (hunk 1 above)
Comment (turrs) [reply src/Parts/MyTrainingCard/SingleTrainingCard.jsx]: thanks for review, this issue already fix here 4a0144443efe14916dc188d008c22c4031fda99a

Comment (turrs) [reply src/Parts/MyTrainingCard/SingleTrainingCard.jsx]: https://www.loom.com/share/f4d62379539d4fa79f9116f2de1ec2e9

**Comment by turrs** [comment]
> Read comments, clean code by remove all "console.log"

thankyou, already remove here 901d037cfd4a273f91d2c348a78940f995c40553

https://www.loom.com/share/820e6fb788b4479cb6ca5c72977faad8

**Pull request merged**